acquia/blt
# Use core's PHPUnit bootstrap file.

Fixes #3091 (sort of)
--------
I don't think it makes sense for BLT to attempt to update sample test files that it distributes, because users probably frequently modify those files, and I don't want to get into the business of munging XML files or something.

Rather, I think we should just try to maintain compatibility across PHPUnit versions, the same way that core does, so that users don't _have_ to update those test files when PHPUnit versions change.

Changes proposed:
---------
- By default, use core's phpunit.xml.dist (and therefore core's bootstrap.php as well), which will alias PHPUnit 4 and 6 class names so that they are interoperable
- Remove our own bootstrap.php and phpunit.xml files, which were either empty boilerplate or copies of core's.

Steps to verify the solution:
-----------
1. Use a BLT project running on PHP 5.6 (necessary to test PHPUnit 4). Apply this patch, and remove tests/phpunit/phpunit.xml. 
2. Install PHPUnit 4. Verify that `blt tests:phpunit` runs.
2. Upgrade to PHPUnit 6 and verify that tests still run.
